### ISSUE TITLE
service/../run_unix blocks on start command

### ISSUE THREAD

**Issue by ciokan** [nextdns/nextdns#156]
`RunOutput` method hangs forever on my EdgeRouter. I managed to fix it by changing the way it reads the command output and by creating a pipe and a goroutine to read from that pipe instead of trying to obtain the output via `stdout, err := cmd.Output()` which seems to be the culprit.

Here's a working code. I didn't do a PR on it because I merely copy pasted bits from [this thread](https://github.com/golang/go/issues/23019) and maybe you feel like having a look/polishing first:

```
func RunOutput(command string, args ...string) (string, error) {
	var stdout string
	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
	defer cancel()
	cmd := exec.CommandContext(ctx, command, args...)
	// obtain a pipe connected to our cmd stdout so we can read
	// from it later. `cmd.Output()` would block here because the
	// underlying pipes are not closed and it will try to read from
	// them indefinitely
	stdoutPipe, err := cmd.StdoutPipe()
	if err != nil {
		cancel()
		return "", fmt.Errorf("%s %s: %w", command, strings.Join(args, " "), err)
	}
	if err = cmd.Start(); err != nil {
		cancel()
		return "", fmt.Errorf("%s %s: %w", command, strings.Join(args, " "), err)
	}
	go func(stdout string, stdoutPipe io.ReadCloser) {
		buf := bufio.NewReader(stdoutPipe)
		for {
			line, _ := buf.ReadString('\n')
			if len(line) > 0 {
				stdout = stdout + line + "\n"
			}
		}
	}(stdout, stdoutPipe)
	err = cmd.Wait()
	if err != nil {
		cancel()
		return "", fmt.Errorf("%s %s: %w", command, strings.Join(args, " "), err)
	}
	return stdout, nil
}
```

The block occurs when running the `start` command

**Comment by rs** [comment]
Are you running the master or the last version?

**Comment by ciokan** [comment]
> Are you running the master or the last version?

master

**Comment by rs** [comment]
This actually revert the fix for #110. Needs some more investigation.
